### PR TITLE
Fixes notification link for comments / sets up jest with TypeScript

### PR DESCRIPTION
**Pull request by broskoski**
Was going to do this all client side, decided to write a test, spent a long time getting jest and enzyme to work with TypeScript, and then realized a better way to solve in the API (here: https://github.com/aredotna/api/pull/1043). 

In any case, tests for TypeScript are setup, will finish and merge this when the API is deployed.

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  roots: ['<rootDir>/src'],
+  transform: {
+    '^.+\\.jsx?$': 'babel-jest',
+    '^.+\\.tsx?$': 'babel-jest',
+  },
```
Comment (damassi) [inline jest.config.js]:
```js
transform: {
  "^.+\\.(ts|tsx|js|jsx)$": "babel-jest",
}
```

File: jest.config.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  testURL: 'http://localhost/',
+  setupTestFrameworkScriptFile: 'jest-enzyme',
```
Comment (damassi) [inline jest.config.js]: This is typically a pointer to a physical file where additional setup can take place. See https://github.com/artsy/force/blob/master/test.config.js

File: package.json [hunk 3]
```diff
@@ -234,6 +240,7 @@
     "sinon": "*",
     "source-map-support": "^0.5.12",
     "storybook-react-router": "^1.0.5",
+    "ts-jest": "^24.0.2",
```
Comment (damassi) [inline package.json]: This should be covered by `babel-jest`, in combination with the `@babel/preset-typescript` (though I imagine this is just experiment code)

**Comment by broskoski** [comment]
Screenshot 2019-05-10 13 19 40

**Pull request merged**